mabl snippets/isElementPresent.js

```diff
@@ -13,9 +13,14 @@ function mablJavaScriptStep(mablInputs, callback) {
     
     // conditional statement to assure that the element is present, as well as
     // assuring the display does not equal 'none'
-    if (element && element.style.display !== 'none') {
-      callback('Element Present');
-    } else {
-      throw Error('Element Not Present');
+    if (element) {
+        const style = window.getComputedStyle(element);
```

Comment:
one thing i just realized is that we should probably test this in a javascript step in a journey just to be sure `window.getComputedStyle` isn't something we block (like `window.alert` and a number of other high-level functionalities).  i don't think it should be, but i know there are a bunch we currently block so worth checking just to be sure.